service/connection.go

```diff
@@ -0,0 +1,33 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+func getConnDB() *mongo.Client {
```

Comment:
```suggestion
func getDBConn() *mongo.Client {
```

What do you think about this name change? I think it makes more sense in English.